core/templates/pages/certificate-assessment-player-page/certificate-assessment-conversation-skin.component.html

```
@@ -1,20 +1,102 @@
+          [checked]="isOptionSelected(option.id)"
+          (change)="toggleMultipleSelect(option.id)">
+        <span class="certificate-assessment-option-text">{{ option.text }}</span>
+      </label>
+    </div>
+
+    <!-- Text / numeric free response -->
+    <div
+      *ngIf="currentQuestion.type === 'text_input' || currentQuestion.type === 'numeric_input'"
+      class="certificate-assessment-free-response">
+      <input
+        [type]="getQuestionInputType()"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)